surveys/2022-wg-debugging-survey/questions.md

```diff
@@ -0,0 +1,360 @@
+# Survey questions
+One of the [compiler team's focuses for
+2022](https://blog.rust-lang.org/inside-rust/2022/02/22/compiler-team-ambitions-2022.html)
+is to improve the experience of debugging Rust code. We'd like to better
+understand your experiences using a debugger with Rust - what's been good,
+what's been bad and how your experience debugging Rust could be better.
+
+Unless you choose to enter your email, your answers will be anonymous. Any
+personal data you submit as a part of this survey will be handled in accordance
+with our policy as described in our Frequently Asked Questions:
+
+https://github.com/rust-community/team/wiki/State-of-the-Rust-Language-Community-Survey-FAQ
```

Comment:
This wiki is out of data (and should probably be removed). Unfortunately, we don't quite have a general privacy policy for surveys, but the one used for the community survey lives [here](https://github.com/rust-lang/surveys/blob/main/documents/data-sharing-annual-community-survey.md).

Reply:
Updated with a link to that document.